packages/flatseal.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Flatseal < Package
+  description 'Flatseal is a graphical utility to review and modify permissions from your Flatpak applications.'
+  homepage 'https://github.com/tchx84/Flatseal'
+  version '1.6.8'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'https://github.com/tchx84/Flatseal/archive/v1.6.8.tar.gz'
+  source_sha256 '28d05995effa2858483283cbc9fb54c55bf4dcb6267e4d7d495a32ff724806b1'
+
+  depends_on 'libhandy'
+  depends_on 'desktop_file_utils'
+  depends_on 'gjs'
+  depends_on 'gtk3'
```

Comment:
I wasn't able to get this working without sommelier so I think we need to add that as a dependency also.

Reply:
Even with GDK_BACKEND=wayland? It worked for me.

Reply:
It's a gtk package so theroretically any gtk package should work

Reply:
I don't think most users will know how to set that up so could you please provide instructions in `def self.postinstall`?

Reply:
It might even be better to create a wrapper script to set `GDK_BACKEND=wayland`.

Reply:
Just kidding, I installed it on a new install without sommelier running and it didn't work.